packages/gestalt/rollup.config.js

```diff
@@ -192,15 +160,20 @@ export default {
         process.env.NODE_ENV || 'development'
       ),
     }),
-    svgPath(),
+    svg(),
     json({
       preferConst: true,
     }),
     babel({
       babelrc: false,
-      presets: [['env', { modules: false }], 'stage-1', 'react'],
+      presets: [
+        ['@babel/preset-env', { modules: false }],
+        '@babel/react',
+        '@babel/flow',
+      ],
+      plugins: ['@babel/proposal-class-properties'],
```

Comment:
Why do we specify these separate from the `babelrc`? Only difference I see is `{ modules: false }`.

Reply:
@btraut yes, leaving this for now. Mostly because afterwards, we compile it down to both a `umd` and a `es` format.

```js
    {
      file: 'dist/gestalt.js',
      format: 'umd',
      name: 'gestalt',
    {
      file: 'dist/gestalt.es.js',
      format: 'es',
      name: 'gestalt',
    },
```